src/containerapp/HISTORY.rst

```diff
@@ -2,9 +2,10 @@
 
 Release History
 ===============
-Upcoming
+0.3.26
```

Comment:
Do you need to release a new extension version for this PR? If so, please also upgrade the version defined in the `setup.py`?